rotkehlchen/tests/api/test_historical_assets_price.py

```diff
@@ -0,0 +1,105 @@
+import random
+from datetime import datetime
+from http import HTTPStatus
+
+import pytest
+import requests
+
+from rotkehlchen.fval import FVal
+from rotkehlchen.tests.utils.api import (
+    api_url_for,
+    assert_error_response,
+    assert_ok_async_response,
+    assert_proper_response_with_result,
+    wait_for_async_task_with_result,
+)
+
+
+@pytest.mark.freeze_time(datetime(2021, 1, 21, 11, 36, 15))
+@pytest.mark.parametrize('assets_timestamp, error_message', [
```

Comment:
same comment as in the previous test for the other endpoint. These are gonna kill our test time and should really be just unit tests.